packages/openai-base/src/tools/image-generation-tool.ts

```
@@ -41,13 +42,20 @@ export function imageGenerationTool(
+      },
     },
-  }
+    'image_generation',
+  )
 }
 
 export { validatePartialImages }
+import {
+  getOpenAIProviderToolMetadata,
+  openAIProviderTool,
+} from './openai-provider-tool'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)